bin/publish-github-release/index.js

```diff
@@ -12,7 +12,7 @@ class GithubReleaseClient {
     this.branch = branch || 'master';
   }
 
-  publishRelease(version, releaseTitle, releaseDescription) {
+  publishRelease(version, releaseTitle, releaseDescription, preRelease) {
```

Comment:
Should it be `prerelease` as it is not two words?

Reply:
aight